styles.css

```diff
@@ -804,30 +963,48 @@
     }
 
     .home-link a {
-        font-size: 16px; 
+        font-size: 16px;
     }
 }
 
 @media (max-width: 480px) {
     .social-container {
         padding: 15px;
-        gap: 16px; 
+        gap: 16px;
     }
 
     .social {
-        gap: 16px; 
+        gap: 16px;
     }
 
     .social a img {
-        width: 16px; 
+        width: 16px;
         height: auto;
     }
 
     .home-link a {
-        font-size: 14px; 
+        font-size: 14px;
     }
 }
 
+/* Media query for desktop  */
+@media (min-width: 769px) {
+    .social-container {
+        flex-direction: row; 
+        justify-content: space-between; 
+        align-items: center; 
+        padding: 40px 60px;
+    }
+
+    .social {
+        justify-content: flex-start; 
+        gap: 32px; 
+    }
+
+    .home-link {
+        justify-content: flex-end;
+    }
+}

```

Comment:
Hi @khadijahMu ,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- Avoid making the website to overflow

[- check](https://www.loom.com/share/775a9ea7ad1e4422b55f84431f4e69e4?sid=e8a8a129-afd1-4dd4-bc16-296b72feab2c).

- Remove the hover effect on all the cards

### 1. Remove the hover effect on each card

![Step 1 screenshot](https://images.tango.us/workflows/ea62b2b5-c97d-4612-beee-40037d0c1c23/steps/84f41706-70a9-4c4f-9c28-c5b8fc6697c9/90acd5a6-8fde-4798-91a9-9d4ab8ec8916.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&w=1200&border=2%2CF4F2F7&border-radius=8%2C8%2C8%2C8&border-radius-inner=8%2C8%2C8%2C8&blend-align=bottom&blend-mode=normal&blend-x=0&blend-w=1200&blend64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmstdjIucG5n&mark-x=5&mark-y=16&m64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL2JsYW5rLnBuZz9tYXNrPWNvcm5lcnMmYm9yZGVyPTQlMkNGRjc0NDImdz0zNjYmaD0zMjcmZml0PWNyb3AmY29ybmVyLXJhZGl1cz0xMA%3D%3D)

### 2. This is how your cards should be display

![Step 2 screenshot](https://images.tango.us/workflows/ea62b2b5-c97d-4612-beee-40037d0c1c23/steps/9e31b014-b506-4101-95bb-c481993139ba/0995e656-e21d-4dd6-aa2b-a88e7320c7f9.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&w=1200&border=2%2CF4F2F7&border-radius=8%2C8%2C8%2C8&border-radius-inner=8%2C8%2C8%2C8&blend-align=bottom&blend-mode=normal&blend-x=0&blend-w=1200&blend64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmstdjIucG5n)

<br/>

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

Reply:
i have updated the required changes. Thankyou!